src/main/java/com/saucedemo/pages/Login.java

```diff
@@ -48,10 +48,10 @@ public String getLoginLockOutError() {
         return lockOutError.getText();
     }
 
-    public void login(String user, String password) {
+    public void login(String username, String password) {
 
         driver.get("https://www.saucedemo.com/");
```

Comment:
In the next pr, you can try and move it inside constructor after 
this.driver = driver; step.